applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/adjoint_linear_strain_energy_response_function.cpp

```diff
@@ -77,7 +77,7 @@ namespace Kratos
     {
         KRATOS_TRY;
 
-        if (rSensitivityMatrix.size2() == 0)
+        if (rSensitivityMatrix.size1() == 0)
```

Comment:
Why does here and below size2 change to size1? Somehow I don't get it.

Reply:
If a design variable is not known to a element / condition the size of the sensitivity matrix will be (size1=0, size2=number of dofs). In order to that we have to check size1.